### PR TITLE
Speed-up of whales pipeline preprocessing steps

### PR DESCRIPTION
**Pull request by pawel-ziecina**
- switch to turbo jpeg (faster loading + initial rescaling)
- rescaling to ~2x of target size to speed-up augmentation

### CODE REVIEW

**Comment by pawel-ziecina** [comment]
Need to update notebooks and installation guide on necessity to install libturbojpeg

**Comment by pawel-ziecina** [comment]
Still testing - dry_train, and then each of tasks. Will inform on results

File: minerva/whales/preprocessing.py [hunk 1]
```diff
@@ -96,33 +99,51 @@ def __init__(self, X, y, img_dirpath, augmentation, target_size, bins_nr):
         self.preprocessing_function = None
         self.normalization_function = None
 
-    def load_image(self, img_name):
-        img_filepath = os.path.join(self.img_dirpath, img_name)
-        return Image.open(img_filepath, 'r')
+    def load_image(self, img_name, max_scale=8):
+        import turbojpeg
```
Comment (kamil-kaczmarek) [inline minerva/whales/preprocessing.py]: @pawel-ziecina why we put this import here?
(@jakubczakon )

File: minerva/whales/preprocessing.py [hunk 2]
```diff
@@ -96,33 +99,51 @@ def __init__(self, X, y, img_dirpath, augmentation, target_size, bins_nr):
...
+        img_path = Path(self.img_dirpath) / img_name
+        img = Image.open(img_path.as_posix())
+
+        scale = 0.5
```
Comment (kamil-kaczmarek) [inline minerva/whales/preprocessing.py]: @pawel-ziecina what is the difference between `max_scale=8` and `scale = 0.5`. Do you think that we should parametrize this -> move it to config file?

(@jakubczakon )

**Comment by kamil-kaczmarek** [comment]
Hey @pawel-ziecina I remember that you mentioned that this piece of work is not done yet. Do you have any estimate when you will have some time to make some progress here?

Thank

Best,
Kamil

**Comment by kamil-kaczmarek** [comment]
@pawel-ziecina thanks for the prompt answer :)

**Pull request merged**